docs/core/deploying/index.md

```diff
@@ -271,6 +288,49 @@ When you deploy your application, any third-party dependencies used in your app
 
 Note that you can only deploy a self-contained app with a third-party library to platforms supported by that library.
 
+## Deploying a Modular Self-Contained App ##
+
+The previous two sections on deploying a self-contained app created and deployed an app that contained the entire .NET Core library. You can also deploy a modular self-contained app that includes only the .NET Core assemblies actually needed by your app.
+
+The word parsing example shown earlier in the [Deploying a Simple Self-Contained App](#simpleSelf)section uses the console, which is defined in the System.Console assembly, and the regular expression engine, which is defined in the System.Text.RegularExpressions assembly. Instead of deploying all of the .NET Standard Library with our app, we can package just the assemblies we need, along with their dependencies and the libraries that contain basic .NET Core types. This requires a modification of the `dependencies` section of our project.json file:
```

Comment:
@svick I think I didn't make this clear -- I meant the dependencies of the assemblies (for example, the dependencies of System.Text.RegularExpression include System.Collections, System.Globalization, System.Threading, etc.) I'll rephrase it as "the assemblies we need, the dependencies of those assemblies, and the libraries that contain basic .NET Core types."